examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java

```diff
@@ -60,7 +60,7 @@
 
   @BeforeClass
   public static void setUp() {
-    PipelineOptionsFactory.register(BigQueryTornadoesITOptions.class);
+    PipelineOptionsFactory.register(TestPipelineOptions.class);
```

Comment:
Ditto

Reply:
I was getting 
```
java.lang.IllegalArgumentException: Property [input] is marked with contradictory annotations. Found [[Default.String(value=gs://apache-beam-samples/shakespeare/) on org.apache.beam.examples.complete.TfIdf$Options#getInput()], [Default.String(value=clouddataflow-readonly:samples.weather_stations) on org.apache.beam.examples.cookbook.BigQueryTornadoes$Options#getInput()]].
```
in test, https://ci-beam.apache.org/job/beam_PostCommit_Java_DataflowV1_PR/61/testReport/org.apache.beam.examples.cookbook/BigQueryTornadoesIT/classMethod/.

Is this a bug for the validation in PipelineOptionsFactory.register?

Reply:
Not a bug. Each PipelineOptions instance must be internally consistent (e.g. types and default values have to be the same). Registering a PipelineOptions makes it such that all PipelineOptions instances have to conform to this new option. It must be that the ITs used to run in different JVMs and now are running in the same JVM.